hazelcast/include/hazelcast/client/ClientConfig.h

```diff
@@ -395,10 +405,10 @@ namespace hazelcast {
 
             std::map<std::string, config::ReliableTopicConfig> reliableTopicConfigMap;
 
-            std::map<std::string, config::NearCacheConfig> nearCacheConfigMap;
+            util::SynchronizedMap<std::string, config::NearCacheConfig> nearCacheConfigMap;
```

Comment:
It is used as ConcurrentHashMap at Java side. If not needed then we can remove after removing from Java implementation.